### PR TITLE
refine make process for #316

### PR DESCRIPTION
**Pull request by tony1223**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -32,3 +32,6 @@
 /config/bbgo.yaml
 
 /pkg/server/assets.go
+
+node_modules
+yarn.lock
```
Comment (c9s) [inline .gitignore:37]: we should include the lock?

File: Makefile [hunk 2]
```diff
@@ -24,8 +24,10 @@ $(BIN_DIR):
 
 
 # build native bbgo
-bbgo:
-	go build -tags web,release -o $(BIN_DIR)/$@ ./cmd/bbgo
+bbgo: static bbgo-full
```
Comment (c9s) [inline Makefile:27]: can be just:

```
bbgo: static
        go build -tags web,release -o $(BIN_DIR)/bbgo ./cmd/bbgo
```

target bbgo is used to build the default bbgo (for local environment)

File: Makefile [hunk 3]
```diff
@@ -147,7 +149,8 @@ docker-push:
 	bash -c "[[ -n $(DOCKER_TAG) ]] && docker push yoanlin/bbgo:$(DOCKER_TAG)"
 
 frontend/out/index.html:
-	(cd frontend && yarn export)
+	cd frontend && yarn install
+	cd frontend && yarn export
```
Comment (c9s) [inline Makefile:153]: you can add a target:

```
frontend/node_modules:
        cd frontend && yarn install

frontend/out/index.html: frontend/node_modules
        cd frontend && yarn export
```

File: .gitignore (hunk 1 above)
Comment (tony1223) [reply .gitignore:37]: yep, it seems you are right. 
https://classic.yarnpkg.com/en/docs/yarn-lock/

**Comment by tony1223** [comment]
I have updated the related files. ccc

**Review by c9s — APPROVED** [review]
thanks!

**Pull request merged**